packages/types/src/documents/app/view.ts

```diff
@@ -35,7 +35,12 @@ export interface View {
 
 export type ViewFieldMetadata = UIFieldMetadata & {
   readonly?: boolean
-  schema?: Record<string, RelationSchemaField>
+  columns?: Record<string, RelationSchemaField>
```

Comment:
Will this cause any problems with existing view metadata? Since anything stored contains the `schema` property?

Reply:
No, this is a new field that has not yet been gone to master